frontend/src/libs/helpers/get-selected-route.ts

```diff
@@ -0,0 +1,16 @@
+import { AppRoute } from '#libs/enums/app-route.enum.js';
+
+type Properties = {
+  pathname: string;
+  routePath: string;
+};
+
+const getSelectedRoute = ({ pathname, routePath }: Properties): boolean => {
```

Comment:
I think this is not a good approach, we should pass needed routes explicitly in routes config